R/summary_functions.R

```diff
@@ -176,6 +232,23 @@ print.coverage <- function(x, group = c("filename", "functions"), by = "line", .
     message(crayon::bold(paste0(names(by_coverage)[i], ": ")),
       format_percentage(by_coverage[i]))
   }
+
+  if (isTRUE(include_branches)) {
+    print_branch_coverage(x, group=group)
+  }
+
+  invisible(x)
+}
+
+#' @export
+print.coverages <- function(x, ...) {
+  for (i in seq_along(x)) {
+    # Add a blank line between consecutive coverage items
+    if (i != 1) {
+      message()
```

Comment:
message prints to stderr while print to stdout - are you use you want this?

Reply:
This segment of code is from the original covr. I did not change it.  I don't know why it looks like I added them.

Reply:
OK